Runtime/Utils/Audio/IAudioData.cs

```diff
@@ -6,7 +7,7 @@ namespace Innoactive.Creator.Core.Audio
     /// This class provides audio data in form of an AudioClip. Which also might not be loaded at the time needed,
     /// check first if there can be one provided.
     /// </summary>
-    public interface IAudioData
+    public interface IAudioData : ICanBeEmpty
```

Comment:
It's sad 😟

Reply:
:(

Reply:
https://media1.giphy.com/media/M28rUlcjueKUE/giphy.gif?cid=ecf05e47ntozs4clmprw4elsbodblyfety0eely22yhkwu14&rid=giphy.gif